src/main/java/gregtech/api/recipes/builders/BlastRecipeBuilder.java

```diff
@@ -32,12 +34,12 @@ public BlastRecipeBuilder copy() {
     }
 
     @Override
-    public boolean applyProperty(String key, Object value) {
+    public boolean applyProperty(@Nonnull String key, Object value) {
         if (key.equals(TemperatureProperty.KEY)) {
             this.blastFurnaceTemp(((Number) value).intValue());
             return true;
         }
-        return true;
+        return super.applyProperty(key, value);
```

Comment:
This is a behavior change. Previously this method would return true, now if there is no cleanroom property in the recipe, this will return false.

Although, that may be fine? I am not sure if this is supposed to return `true` regardless of if the property was applied, or false like all the other specialized builders.